### PR TITLE
FIX #238

### LINKED ISSUE #238: In posts leaderboard table layout is not fixed
http://gistflow.com/posts/leaderboard?page=2

If post has a lot of tags width gets broken.

### PR DESCRIPTION
**Pull request by makaroni4**

### CODE REVIEW

**Comment by releu** [comment]
Why `float: left` ?

**Comment by makaroni4** [comment]
`display: inline`:



`float: left`:

File: app/assets/stylesheets/sections/leaderboard.css.scss [hunk 1]
```diff
@@ -19,20 +19,21 @@ section.leaderboard {
...
     }
 
     div.tags {
+      width: 300px;
       ul {
         li {
-          display: inline;
-          margin-right: 3px; 
+          margin-right: 3px;
+          float: left;
```
Comment (releu) [inline app/assets/stylesheets/sections/leaderboard.css.scss]: try to use `display: inline-block` without width.

**Comment by makaroni4** [comment]
:+1:

**Pull request merged**